code/datums/components/crafting/crafting.dm

```diff
@@ -12,47 +12,14 @@
 	CL.screen += C
 	RegisterSignal(C, COMSIG_CLICK, PROC_REF(component_ui_interact))
 
+#define COOKING TRUE
+#define CRAFTING FALSE
```

Comment:
do these really need to have the value TRUE/FALSE?
Just go from 1, 2, etc

Reply:
I will change it to numbers if there will be a third mode. It's just easier to check with bool.

Reply:
Like here:

```dm
if (ispath(recipe.type, /datum/crafting_recipe/food/) != mode) // Skip if food and mode is crafting / Skip if not food and mode is cooking
```